BitPay/Exceptions/SettlementException.cs

```diff
@@ -14,5 +15,19 @@ public SettlementException() : base(BitPayCode, BitPayMessage)
         public SettlementException(Exception ex) : base(BitPayCode, BitPayMessage, ex)
         {
         }
+
+        public SettlementException(string bitpayCode, string message) : base(bitpayCode, message)
```

Comment:
Is this one neccesary?

Reply:
I don't think, But i added because we have same method in others exceptions.

Reply:
in other you have only 2 constructors

Reply:
Yes, there are 2 constructors but those are in crud exceptions  of models.
Parent Constructors of every model has more then 2 constructors.